kong/api/api_helpers.lua

```diff
@@ -257,6 +257,8 @@ local function parse_params(fn)
 
     local res, err = fn(self, ...)
 
+    kong.worker_events.poll()
```

Comment:
even if `fn()` returned an error?

Reply:
@javierguerragiraldez, do you mean it would be beneficial to log it with e.g. `NOTICE`?

Reply:
Ok, now I understand. Yes, I thought about it. At some point I had check whether `method` was `POST|PUT|PATCH|DELETE`, but removed it. The poll in most cases is fast. Because we don't know where `fn()` fails, I just decided to call it always.